app/javascript/packs/containers/OptionsContainer.jsx

```diff
@@ -9,28 +9,34 @@ import Question from '../components/Question';
 import { connect } from 'react-redux';
 import { getAnswerForQuestionAndBuilding } from '../selectors/answersSelector';
 import { getDependentQuestionsForOptions } from '../selectors/questionsSelector';
-import { addAnswer, editAnswer } from '../actions/answers';
+import { createAnswer, updateAnswer } from '../actions/answers';
 
 class OptionsContainer extends React.Component {
-  handleSelect(option_id, text) {
+  /**
+   * Callback for when an option has been triggered. Will update the answer in database and store.
+   *
+   * @param option_id   id of the selected option
+   * @param value       The data of the updated answer. For range options, this is the number the user inputted;
+   *                      for dropdown options, it's the text of the option that was selected.
+   */
+  handleSelect(option_id, value) {
     // Set up answer data to send in fetch request
+    console.log("ANSWER");
```

Comment:
Do we want to remove these log statements before merging?

Reply:
Oh oops! Just removed them